README.md

```diff
@@ -12,26 +12,85 @@ $ mkdir "$HOME/data_server/results" -p
 
 ## Usage
 
-Use the host network, `--net=host`, for traffic to get from a remote client through the host to the container. The bind mount volume is labeled as a shared directory with the host, `:z`, to persist results and facilitate integration with other tools on the host.
+### Overview
+
+1. Create a means of communication between the containers.
+    * Podman containers can easily communicate with `localhost` when they're in the same pod.
+    * Docker containers can easily communicate over the default `bridge` network when each service is given a `network-alias`.
+
+2. Start a Postgresql database service.
+
+3. Edit the environment variables with appropriate usernames and passwords. Run the application deployment's `prestart` to seed the database with user accounts.
```

Comment:
What do you mean by appropriate? Should I pick something fun or is this a predefined user/pass?

Reply:
Only that username has to be a valid email regular expression.